integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDomainInPVUsingWDT.java

```diff
@@ -73,12 +73,11 @@ class ItFmwDomainInPVUsingWDT {
   private static final String RCUSCHEMAUSERNAME = "myrcuuser";
   private static final String RCUSCHEMAPASSWORD = "Oradoc_db1";
 
-  private static final String DOMAINHOMEPREFIX = "/shared/" + domainNamespace + "/domains/";
-
   private static String dbUrl = null;
   private static LoggingFacade logger = null;
 
   private static final String domainUid = "fmw-domainonpv-wdt";
+  private static final String DOMAINHOMEPREFIX = "/shared/" + domainUid + "/domains/";
```

Comment:
can we include the namespace as well in the path to make it unique?

Reply:
done